src/groupeddataframe/splitapplycombine.jl

```diff
@@ -499,25 +512,40 @@ wrap(x::AbstractMatrix) =
     NamedTuple{Tuple(gennames(size(x, 2)))}(Tuple(view(x, :, i) for i in 1:size(x, 2)))
 wrap(x::Any) = (x1=x,)
 
-wrap_table(x::Any) =
+wrap_table(x::Any, ::Val) =
     throw(ArgumentError("return value must not change its kind " *
                         "(single row or variable number of rows) across groups"))
-wrap_table(x::Union{NamedTuple{<:Any, <:Tuple{Vararg{AbstractVector}}},
-                    AbstractDataFrame,
-                    AbstractVector,
-                    AbstractMatrix}) = wrap(x)
+function wrap_table(x::Union{NamedTuple{<:Any, <:Tuple{Vararg{AbstractVector}}},
+                             AbstractDataFrame, AbstractMatrix}, ::Val{T}) where T
```

Comment:
Why not call `T` `wasmulti` as below? `T` evokes a type, and it's very hard to follow the logic when you don't know what it means. :-)

Reply:
ok